tools/common/schema/updatetask.go

```diff
@@ -30,15 +30,13 @@ import (
 	// should not be used for anything more important (password hashes etc.). Marking it as #nosec because of how it's
 	// being used.
 	"crypto/md5" // #nosec
-	"embed"
```

Comment:
i think the idea is to embed the schema into the tool's binary so the schema become optional at runtime, but you can still provide your schema if you want to otherwise it will use the embedded version. 
Would this PR break that logic?

Reply:
I just gave it a try.
I found that actually embedded schema cannot not be used in windows because the embed filesystem uses "/" while the allowed values of the `--schema-name` cli option is "\" on windows

It gives this error on windows
```
Config Error:(-schema-name) must be one of: [mysql\\v8\\temporal mysql\\v8\\visibility postgresql\\v12\\temporal postgresql\\v12\\visibility]
```

Do you want to the include a fix for this as well in the PR?

Reply:
Yes, we should fix that, I think we want to keep the embed schema working.

Reply:
@yiminc 
Fixed in `embed.go`
I updated the readme as well. TBH I didn't know the schema is embedded until you told me.